core/src/com/unciv/ui/saves/LoadGameScreen.kt

```diff
@@ -138,6 +144,52 @@ class LoadGameScreen(previousScreen:BaseScreen) : PickerScreen(disableScroll = t
         return rightSideTable
     }
 
+    private fun handleLoadGameException(primaryText: String, ex: Exception?) {
+        var errorText = primaryText.tr()
+        if (ex is UncivShowableException) errorText += "\n${ex.message}"
+        errorLabel.setText(errorText)
+        ex?.printStackTrace()
+        if (ex is MissingModsException) {
+            loadMissingModsButton.isVisible = true
+            missingModsToLoad = ex.missingMods
+        }
+    }
+
+    private fun loadMissingMods() {
+        loadMissingModsButton.isEnabled = false
+        descriptionLabel.setText("Loading...".tr())
+        crashHandlingThread(name="DownloadMods") {
+            try {
+                val mods = missingModsToLoad.replace(' ', '-').lowercase().splitToSequence(",-")
+                for (modName in mods) {
+                    val repos = Github.tryGetGithubReposWithTopic(10, 1, "$modName+") // + is important here!
```

Comment:
`+` why? ... I see. OK that should be the responsibility of the function not the caller.